src/Mod/Fem/App/AppFemPy.cpp

```diff
@@ -263,11 +263,11 @@ class Module : public Py::ExtensionModule<Module>
         {
             App::Document* pcDoc = App::GetApplication().getActiveDocument();
             App::DocumentObject* obj = pcDoc->getObject(resName.c_str());
-            FemVTKTools::readFluidicResult(EncodedName.c_str(), obj);
+            FemVTKTools::readResult(EncodedName.c_str(), obj);
         }
         else
-            FemVTKTools::readFluidicResult(EncodedName.c_str());  //assuming activeObject can hold Result
-        
+            FemVTKTools::readResult(EncodedName.c_str());  //assuming activeObject can hold Result
```

Comment:
for uniformity sake adding a whitespace to beginning of comment 
`// assuming activeObject can hold Result`